packages/grid/_modules_/grid/models/gridExport.ts

```diff
@@ -1,12 +1,24 @@
+export interface GridExportCsvOptions {
+  utf8WithBom?: boolean;
+}
+export interface GridExportFormatCsv {
+  format: 'csv';
+  options?: GridExportCsvOptions;
+}
+
 /**
  * Available export formats. To be extended in future.
  */
-export type GridExportFormat = 'csv';
+export type GridExportFormatExtension = 'csv';
```

Comment:
Ths is a breaking change, do we really need it?

Reply:
I have reverted this change back.